bindings/nodejs/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    env: {
+        node: true,
+        browser: true,
```

Comment:
Not sure if we need to set it true?

Reply:
I've looked into it and I can't find a reason we need this, no.